src/sagas/ses.js

```diff
@@ -0,0 +1,102 @@
+/**
+ * Copyright (c) Hathor Labs and its affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+import { MMKV } from 'react-native-mmkv';
+import RNRestart from 'react-native-restart';
+import {
+  call,
+  fork,
+  take,
+  all,
+} from 'redux-saga/effects';
+import { checkForFeatureFlag } from './helpers';
+import { SES_FEATURE_TOGGLE, SHOULD_ENABLE_SES_STORAGE_KEY } from '../constants';
+
+const storage = new MMKV();
+
+function verifySesEnabled() {
+  return Object.isFrozen([].prototype);
+}
+
+function disableSes(restart = true) {
+  console.log('disabling SES!!');
+  storage.set(SHOULD_ENABLE_SES_STORAGE_KEY, false);
+
+  if (restart) {
+    RNRestart.restart();
+  }
+}
+
+function enableSes(restart = true) {
+  storage.set(SHOULD_ENABLE_SES_STORAGE_KEY, true);
+
+  if (restart) {
+    RNRestart.restart();
```

Comment:
question: Should we expect any negative side effects from a React Native restart? Any notes on how we should organize our code not to trigger those side effects by accident?

At a quick glance over the React Native Restart docs I found nothing noteworthy, but I think it's a question important enough to be brought up.

Reply:
I think there are negative side effects in usability, as if this happens while the user is doing something like creating a token or sending a transaction, the app will restart and he will lose his progress.. But I think this is acceptable 

We don't need to do this though, if we just remove this restart, SES will be enabled the next time he opens the app..